creation/web_base/singularity_lib.sh

```diff
@@ -1647,6 +1647,14 @@ singularity_get_image() {
         [[ -z "$singularity_image" ]] && singularity_image=$(dict_get_first SINGULARITY_IMAGES_DICT)
     fi
 
+    if [[ -n "$CVMFS_MOUNT_DIR" ]]; then
+        # set things up here since the path needs to be bindmounted inside the container
+        local mount_home=${CVMFS_MOUNT_DIR/\/dist\/cvmfs/}
+        local symlink_target
+        symlink_target=$(readlink $mount_home/dist/${singularity_image#/})
+        singularity_image=$mount_home/dist/${symlink_target#/}
```

Comment:
discussed about this and decided to remove the changes pertaining to singularity invocation with on demand cvmfs and include as a patch to the production series. this was decided since singularity invocation handling needs to be ensured irrespective of whether CVMFS is locally available where it could be mounted at /cvmfs or /opt/cvmfs or CVMFS is provisioned via cvmfsexec using modes 1-3.